### DISCUSSION TITLE
Fighting Grace

### DISCUSSION THREAD

**Discussion by LouderthanThunder** [Lambosaurus/hives-and-torches#27] [Feats]
You may expend a single crit to move 5 foot ignoring free strikes and contested zones. This movement replaces the crit damage

**Comment by Lambosaurus** [comment]
I like this idea a lot. I wonder if it should also give some minor passive effect? Otherwise it may feel very situational.

I'll reword it like:

New critical option:
 - You may make an immediate manoeuvre action.

**Reply by Hoppelite** [reply]
This would be cool, because then you could do thing like stand from prone (assuming there is a time when you are prone that you can crit)
